tests/test_toolchain.py

```diff
@@ -74,16 +85,54 @@ def test_create(self):
         assert m_get_toolchain_versions.call_args_list == [
             mock.call('/tmp/android-ndk', mock.ANY)]
         assert m_get_cython_path.call_args_list == [mock.call()]
-        assert m_build_dist_from_args.call_count == 1
+        build_order = [
+            'hostpython3', 'libffi', 'openssl', 'sqlite3', 'python3',
+            'genericndkbuild', 'setuptools', 'six', 'pyjnius', 'android',
+        ]
+        python_modules = []
```

Comment:
maybe put the `[]` as a direct parameter into the `mock.call` and add ` #python_modules parameter` comment instead? I assume you put this separately for readability, but it did throw me off at first that this is so weirdly put separate

Reply:
Yes this is exactly for readability, I specifically prefer explicit code rather than a comment.
I mean what you say is also fine and I know people do it and then if I were to be consistent I should do the same for at least `context` and `project_dir` as well right? Then the code would look something like:
```python
assert m_build_recipes.call_args_list == [
    mock.call(
        build_order,
        # python_modules parameter
        [],
        # context parameter
        mock.ANY,
        # project_dir parameter
        None,
        ignore_project_setup_py=False
    )
]
```
I honestly don't like it at all :confused: but if you definitely prefer comment than code in this case, I can still had it. And what about `build_order` would you also prefer literal with a comment? Or perhaps I misunderstood what you're suggesting :thinking: ?

Reply:
yes you got my gist! what about put just a little differently:

```
assert m_build_recipes.call_args_list == [
    mock.call(
        build_order,
        [],                # python_modules
        mock.ANY,          # context
        None,              # project_dir
        ignore_project_setup_py=False
    )
]
```
Something like that?

I am only suggesting something different because at first glance the separately written block made me think there was something else going on in addition, so it did throw me off for a second. I think putting it as a comment like that does the same job while avoiding that

But as usual whatever you prefer, was just what I personally thought with my occasionally weird style preferences :smile:

Reply:
OK I understand what you say I think and thanks for suggesting, but yes I definitely prefer to use code rather than comments in this case.
I hope you're fine that I keep it this way then